odoo/modules/loading.py

```diff
@@ -330,6 +330,7 @@ def load_modules(db, force_demo=False, status=None, update_module=False):
     models_to_check = set()
 
     with db.cursor() as cr:
+        cr.execute("SET SESSION lock_timeout = '15s'")
```

Comment:
perhaps a small comment to remind the reader why this is safe?

```suggestion
        # prevent endless wait for locks during schema alterations, all timeouts
        # are reset automatically when the connection is released to the pool
        cr.execute("SET SESSION lock_timeout = '15s'")
```